README.md

```diff
@@ -247,3 +247,71 @@ If you have any questions or comments about this component, or need help using i
 ## Licence
 
 This software is published by the Financial Times under the [MIT licence](http://opensource.org/licenses/MIT).
+
+
+
+
+
+
+
+### PASS
+**background: paper**
+	AAA:
+	- black
+	- claret
+	- oxford
+	- slate
+	- velvet
+	AA
+	- teal
+	- crimson
+	- org-b2c-dark
+	AA18
```

Comment:
Does this means these colours pass the WCAG AA rating when the text is sized at 18pt?

Reply:
That wasn't supposed to be in the readme 🤦‍♀️